docs/installation/centos.md

```diff
@@ -41,66 +38,17 @@ packages.
 
 ## Install
 
-There are two ways to install Docker Engine.  You can use `curl` with the  `get.docker.com` site. This method runs an installation script which installs via the `yum` package manager. Or you can install with the `yum` package manager directly yourself.
-
-### Install with the script
-
-
-1. Log into your machine as a user with `sudo` or `root` privileges.
-
-2. Make sure your existing yum packages are up-to-date.
-
-		$ sudo yum update
-		
-3. Run the Docker installation script.
-		
-		$ curl -sSL https://get.docker.com/ | sh
-		
-		This script adds the `docker.repo` repository and installs Docker.
-
-4. Start the Docker daemon.
-
-		$ sudo service docker start
-
-5. Verify `docker` is installed correctly by running a test image in a container.
+There are two ways to install Docker Engine.  You can install with the `yum` package manager directly yourself. Or you can use `curl` with the  `get.docker.com` site. This second method runs an installation script which installs via the `yum` package manager.
```

Comment:
You can .... yourself; "yourself" seems a bit redundant 

(also, could you wrap to 80-chars, it's easier to put my comment on the right part <3)